src/data/programs/nv_programs.ts

```diff
@@ -1,10 +1,18 @@
 export const NV_PROGRAMS = {
-  nv_nVEnergy_homeEnergySaver: {
+  nv_nVEnergy_residentialAppliancesAndProducts: {
     name: {
-      en: 'Home Energy Saver',
+      en: 'Residential Appliances and Products',
```

Comment:
Could you put `NV Energy` at the start of the all the NV Energy program names? (See the last bullet of [this post](https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1711991416542819?thread_ts=1711991416.542819) for why)